src/components/FutureTripCard.js

```diff
@@ -1,22 +1,34 @@
 import styled from 'styled-components';
 import { FiTrash } from 'react-icons/fi';
+import Modal from './Modal';
+import { useState } from 'react';
 
-export default function FutureTripCard({ trip, onDeleteCard }) {
+export default function FutureTripCard({ trip, onDelete, id }) {
   var dayjs = require('dayjs');
   dayjs().format();
 
+  const [visible, setVisible] = useState(false);
+
   return (
-    <Card>
-      <Date>
-        {dayjs(trip.startDate).format('DD-MM-YY')} <span> to </span>
-        {dayjs(trip.endDate).format('DD-MM-YY')}
-      </Date>
-      <Destination>{trip.destination}</Destination>
-      <Notes>{trip.notes}</Notes>
-      <Delete onClick={() => onDeleteCard(trip._id)}>
-        <FiTrash size={25} />
-      </Delete>
-    </Card>
+    <>
+      <Card>
+        <Date>
+          {dayjs(trip.startDate).format('DD-MM-YY')} <span> to </span>
+          {dayjs(trip.endDate).format('DD-MM-YY')}
+        </Date>
+        <Destination>{trip.destination}</Destination>
+        <Notes>{trip.notes}</Notes>
+        <Delete onClick={() => setVisible(!visible)}>
+          <FiTrash size={25} />
+        </Delete>
+      </Card>
+      {visible && (
+        <Modal
+          onDelete={() => onDelete(id)}
```

Comment:
is this the same id as _id? 
If so, you should stay consistent and name all the ids the same :))